### PR TITLE
Adding new page under "Settings" Tab called "Average Peer Review Scores"

### LINKED ISSUE #5: [SUBTASK] Adding a new tab in the "Settings" page called Average Peer Review Scores
**What feature is this a subtask of**
#3 

**What will be implemented in this subtask**
This feature is a frontend implementation. 
  
This feature will be suitably complete when a user can navigate to the Settings page, click on the "Average Peer Reviews" tab on the left hand side and they see a blank page. In addition, a user should be able to generally navigate in and out of this page by clicking in and out from other tabs and also out of the settings page. 

**What files will be created/modified**
- en.json: within the "settings" object that lists all the tabs.
- app.js: adding a state for the settings/review_score page while attributing a template html doc and a ModuleJs controller to that state.
- Create SettingsReviewScore.js for this controller
- settings.html: added a tab for the review scores
- Created settings.review_scores.html: This is the html page that is shown on the Average Review Scores page

**What dependencies does this issue have or create**
This issue doesn't depend on any other issues. Until this issue is finished, the issue of creating a chart to display the average peer reviews (see #8)  can't be started.

**What is the estimated effort of this subtask**
Approximately 1 hour

### PR DESCRIPTION
**Pull request by nhadlaw**
## Issue
Issue resolves #5 

## Description
The purpose of this pull request is to add changes made to the frontend component of the Settings page to display the feature of a new page called "Average Peer Review Scores" that will be used to display a chart of average peer review scores for all users.

## Files changed
- en.json: within the "settings" object that lists all the tabs.
- app.js: adding a state for the settings/review_score page while attributing a template html doc and a ModuleJs controller to that state.
- Create SettingsReviewScore.js for this controller
- settings.html: added a tab for the review scores
- Created settings.review_scores.html: This is the html page that is shown on the Average Review Scores page

## Notes for reviewers
- Look for good code styling
- Look for good variable/function names

## Checklist
Fill out this checklist before merging the PR:
- [x] Created issue
   - [x] Issue has a relevant labels
   - [x] Issue added to the kanban board with its status as In-Progress
   - [x] Issue added to the relevant milestone
   - [x] Issue mentioned in the first section of this PR description
- [x] Testing
   - [x] Added new tests for the changes (or modified existing tests)
   - [x] Passed most recent round of CI
- [x] Review
   - [x] Received first round of review comments
   - [x] Addressed first round of review comments
   - [x] Received second round of review comments
   - [x] Addressed second round of review comments

### CODE REVIEW

File: docs-web/src/main/webapp/src/locale/en.json [hunk 1]
```diff
@@ -281,7 +281,10 @@
     "menu_ldap": "LDAP authentication",
     "menu_metadata": "Custom metadata",
     "menu_monitoring": "Monitoring",
-    "average_peer_review_score": "Average Peer Review Scores",
+    "menu_avg_review_score": "Average Review Scores",
```
Comment (aruiz2) [inline docs-web/src/main/webapp/src/locale/en.json]: "menu_average_review_score" might be better?

File: docs-web/src/main/webapp/src/partial/docs/settings.html [hunk 2]
```diff
@@ -22,6 +22,7 @@
         <a class="list-group-item" ui-sref-active="{ active: 'settings.inbox.**' }" href="#/settings/inbox">{{ 'settings.menu_inbox' | translate }}</a>
         <a class="list-group-item" ui-sref-active="{ active: 'settings.ldap.**' }" href="#/settings/ldap">{{ 'settings.menu_ldap' | translate }}</a>
         <a class="list-group-item" ui-sref-active="{ active: 'settings.monitoring.**' }" href="#/settings/monitoring">{{ 'settings.menu_monitoring' | translate }}</a>
+        <a class="list-group-item" ui-sref-active="{ active: 'settings.review_score.**' }" href="#/settings/review_score">{{ 'settings.menu_avg_review_score' | translate }}</a>
```
Comment (aruiz2) [inline docs-web/src/main/webapp/src/partial/docs/settings.html]: 'settings.menu_average_review_score'

File: docs-web/src/main/webapp/src/app/docs/controller/settings/SettingsReviewScore.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+/**
+ * Settings monitoring controller.
+ */
+angular.module('docs').controller('SettingsReviewScore', function($scope, Restangular) {
+  Restangular.one('app').get().then(function(data) {
+    $scope.app = data;
+    console.log('scope is', $scope)
```
Comment (aruiz2) [inline docs-web/src/main/webapp/src/app/docs/controller/settings/SettingsReviewScore.js]: no semicolon needed at the end of line?

File: docs-web/src/main/webapp/src/app/docs/controller/settings/SettingsReviewScore.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  });
+
+
```
Comment (aruiz2) [inline docs-web/src/main/webapp/src/app/docs/controller/settings/SettingsReviewScore.js]: nit: delete empty line

File: docs-web/src/main/webapp/src/app/docs/controller/settings/SettingsReviewScore.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+/**
+ * Settings monitoring controller.
+ */
+angular.module('docs').controller('SettingsReviewScore', function($scope, Restangular) {
+  Restangular.one('app').get().then(function(data) {
+    $scope.app = data;
+    console.log('scope is', $scope)
+  });
+
```
Comment (aruiz2) [inline docs-web/src/main/webapp/src/app/docs/controller/settings/SettingsReviewScore.js]: nit: delete empty line

File: docs-web/src/main/webapp/src/partial/docs/settings.review_scores.html [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <head>
+
```
Comment (aruiz2) [inline docs-web/src/main/webapp/src/partial/docs/settings.review_scores.html]: nit: remove empty line

File: docs-web/src/main/webapp/src/locale/en.json (hunk 1 above)
Comment (SiddharthParatkar) [reply docs-web/src/main/webapp/src/locale/en.json]: Avg is fine to me, it usually isn't an overloaded shortening

**Review by SiddharthParatkar — APPROVED** [review]
LGTM!

**Review by aruiz2 — APPROVED** [review]

**Pull request merged**